solr/core/src/java/org/apache/solr/handler/PingRequestHandler.java

```diff
@@ -133,6 +134,12 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   public static final String HEALTHCHECK_FILE_PARAM = "healthcheckFile";
+
+  @Override
+  public Name getPermissionName(AuthorizationContext request) {
+    return Name.HEALTH_PERM;
```

Comment:
Thinking about this - should we rather use `metrics-read` here instead of inventing a new permission? Health status is one type of metric.. Only reason to distinguish is if you simply want to allow an external Load Balancer or something to know whether Solr is alive, but don't want to leak all metrics...

Reply:
![Skjermbilde 2021-11-05 kl  18 07 25](https://user-images.githubusercontent.com/409128/140550515-69de8b27-c54d-4a25-85cb-ac31531920cf.png)
From the javadoc of PingRequestHandler... This is a write operation, so perhaps `action=enable` and `action=disable` should be guarded by `config-edit`? Else, it can be used as an attack to disconnect all solr nodes from a Load Balancer :)

Reply:
good catch!

Reply:
I aded a test for "action" as proposed, and just tested locally that it works.